### ISSUE TITLE
BitProgressIndicator component missing features

### ISSUE THREAD

**Issue by maryamhdr** [bitfoundation/bitframework#806]
Some HTML attributes like `id`, `aria-valuemax` and `aria-valuenow`  should be added to the component.
We should add proper class names to the component elements so that the customization would be easier for consumers.
Test methods of the component should be modified based on changes.
